csrankings-b.csv

```diff
@@ -362,6 +362,7 @@ Bernhard Westfechtel,University of Bayreuth,http://www.ai1.uni-bayreuth.de/en/te
 Bernhard Wicht,University of Hannover,https://www.ims.uni-hannover.de/wicht.html?&no_cache=1,xfI3NpoAAAAJ
 Bernie Tiddeman,Aberystwyth University,https://www.aber.ac.uk/en/cs/staff-profiles/listing/profile/bpt,pQHTi7UAAAAJ
 Bernt Schiele [INF],Max Planck Society,https://www.mpi-inf.mpg.de/departments/computer-vision-and-multimodal-computing/people/bernt-schiele,z76PBfYAAAAJ
+Berrak Sisman,SUTD,https://istd.sutd.edu.sg/people/faculty/berrak-sisman,inTpYLkAAAAJ

```

Comment:
This home page redirects to a talk announcement, and the faculty member is not listed [on the dept. website](https://istd.sutd.edu.sg/people/faculty/).

Reply:
This faculty has very recently departed from SUTD, sorry for the outdated info, I have removed her in the new commit.